ext/standard/mail.c

```diff
@@ -57,7 +57,7 @@
 
 extern zend_long php_getuid(void);
 
-static bool php_mail_build_headers_check_field_value(zval *val)
+static bool php_mail_build_headers_check_field_value(zval *val, php_mail_header_value_error_type *error_type)
```

Comment:
this fix should target PHP 8.2 probably

Reply:
My opinion is agree that this PR target is PHP 8.2. What do you think?

Reply:
Indeed, I can't imagine any users being bothered by this change.

Reply:
However, I would like some more opinions on whether this is a change that should be made in the patch version.

Reply:
@ramsey Do you think that this PR target is PHP 8.2? I would like to hear the opinion of the release manager.

Reply:
it should be fine as a fix - please target 8.2